src/Pages/Auth/Register.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import logo from "../../assets/budget-it.png";
+import { currency } from "./currency";
+const Register = () => {
+  const [user, setUser] = useState({
+    firstname: "",
+    lastname: "",
+    email: "",
+    password: "",
+    confirmPassword: "",
+    currency: "NGN",
+  });
+
+  const handleChange = (e) => {
+    setUser({
+      ...user,
+      [e.target.name]: e.target.value,
+    });
+  };
+  const handleRegSubmit = (e) => {
+    e.preventDefault();
+  };
+  return (
+    <div className="auth">
+      <div className="side">
+        <img src={logo} alt="BudgetIt logo" />
+        <h3> Enter your details</h3>
+        <span> Budget your spending.</span>
+        <span>Budgeting never felt so good.</span>
```

Comment:
```suggestion
        <span><em>Budgeting never felt so good....</em></span>
```

Reply:
Can you also animate them to display like as if you are typing?

Reply:
Ohk